Templates/CSharp/Requests/MethodRequest.cs.tt

```diff
@@ -298,7 +298,7 @@ namespace <#=method.Namespace.GetNamespaceName()#>
         /// </summary>
         /// <param name="<#=returnEntityParameter#>">The <#=returnEntityType#> object set with the properties to update.</param>
         /// <returns>The task to await for async call.</returns>
-        public <#=methodReturnType#> PatchAsync(<#=returnEntityType#> <#=returnEntityParameter#>)
+        public <#=methodOverloadReturnType#> PatchAsync(<#=returnEntityType#> <#=returnEntityParameter#>)
```

Comment:
These two variables seem to have the same value throughout the file (assignment at line 33)

Reply:
They are the same unless the return type is a collection.